packages/financial-templates-lib/src/clients/ExpiringMultiPartyClient.js

```diff
@@ -117,10 +117,11 @@ class ExpiringMultiPartyClient {
       await this.initialSetup();
     }
     // Fetch contract state variables in parallel.
+    const fromBlock = await getFromBlock(this.web3);
```

Comment:
@UMAprotocol/eng Where else in the repo can we take advantage of this function and reduce web3 calls?